7/debian-10/rootfs/opt/bitnami/scripts/logstash/run.sh

```diff
@@ -26,6 +26,10 @@ else
     args=( "-f" "$LOGSTASH_CONF_FILE" )
 fi
 
+if [[ -n "$LOGSTASH_EXTRA_ARGS" ]]; then
+    args+=( $LOGSTASH_EXTRA_ARGS )
+fi
```

Comment:
Let's put this block below the line about exposing the API, so that the extra args are the last to be added.

Also, I will suggest this change to make it similar to Keycloak:

```suggestion
if [[ -n "$LOGSTASH_EXTRA_ARGS" ]]; then
    read -r -a extra_args <<<"$LOGSTASH_EXTRA_ARGS"
    args+=("${extra_args[@]}")
fi
```

Reply:
Good poinit. I'll make this change.